src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -754,3 +758,79 @@ impl RustcTargetData {
         self.target_config(kind).links_overrides.get(lib_name)
     }
 }
+
+/// Structure used to deal with Rustdoc fingerprinting
+#[derive(Debug, Serialize, Deserialize)]
+pub struct RustDocFingerprint {
+    _rustc_verbose_version: String,
```

Comment:
The leading `_` here seems a little unusual.

Reply:
I added it because it's never used per se. But ye, might be better to remove it.